src/routers/router.ts

```diff
@@ -1,16 +0,0 @@
-import { Router } from 'express';
-import authRouter from './authRouter.js';
-import cardRouter from './cardRouter.js';
-import credentialRouter from './credentialRouter.js';
-import noteRouter from './noteRouter.js';
-import wifiRouter from './wifiRouter.js';
-
-const router = Router();
-
-router.use(authRouter);
-router.use(cardRouter);
-router.use(credentialRouter);
-router.use(noteRouter);
-router.use(wifiRouter);
```

Comment:
Você pode deixar com 2 parâmetros esses uses, assim você deixar como padrão, o início do caminho das rotas aqui, e não dentro dos routers, gerando menos repetição